samples/ibm-samples/ffdl-seldon/credentials/github-creds.ini

```diff
@@ -0,0 +1,3 @@
+[CREDENTIALS]
+config_file_url = https://raw.githubusercontent.com/Tomcli/kfp-secrets/master/creds.ini
```

Comment:
The github token is sensitive information and should ideally not be checked in.

Reply:
This is dummy token